packages/utils/navigation.ts

```diff
@@ -106,6 +111,11 @@ const navConfig: {
     Governance: "governance",
     UserPublicProfile: "user/:id/:tab?",
     RiotersFooter: "rioters-footer",
+    AdministrationDashboard: "administration-dashboard",
+    LaunchpadApplications: "launchpad-applications",
+    ApplicationReview: "application-review",
+    ReadyLaunchpadApplications: "ready-launchpad-applications",
+    AllProjectAdministrationDash: "all-project-applications-dashboard",
```

Comment:
these should be shorter and under a common path, something like
- `launchpad/admin`
- `launchpad/applications`
- etc..

Reply:
changed, PR updated